### DISCUSSION TITLE
Welcome to the Lua Node Editor Discussions! 👋

### DISCUSSION THREAD

**Discussion by SanderKofficial** [SanForgeStudio/LuaNodeEditor#19] [Announcements]
Welcome to the discussions tab, here everyone can open discussions and talk about the Lua Node Editor, 

Every bit of feedback, ideas, suggestions etc is highly appreciated and will be part of our development because we wanna help the overall developers and learning communities the best as we can.

Thank you all ❤️

**Comment by r2ks** [comment]
is this Project Dead or still active?

**Reply by SanderKofficial** [reply]
Well that kind of depends, i personally am not actively working on it, but if there are some features or certain tools that you would need then i can look into that ofcourse.

**Reply by r2ks** [reply]
Thank you for the response is the library up to LUA 5.4 This is so amazing is it stable enough to use ??
